### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by joelbibby** [precisionpete/wgadmin#1] [General]
Hi there, how do we access the UI? There is no instruction in the readme.

Thanks.

**Comment by joelbibby** [comment]
When trying my ip:8000, nothing loads. So I changed the JSON file to my ip: 192.168.1.250:8000, restarted service, but still doesn't load the UI

HttpListen
The IP and Port running the Web GUI. Defaults to 0.0.0.0:8000. Note: 0.0.0.0 means listen on all interfaces. The app runs as root so ports below 1024 are possible if they are free. e.g. 80

This is with the manual no-debian install.

With the packaged version, it says Wireguard is not installed, it definitely is. (Also, the file name in your readme, for 32 bit packaged, says 64)

Cheers.

**Reply by precisionpete** [reply]
Is it possible you have a firewall running on that machine that is blocking the port? You will need to open the port.

Is it possible you already have a web server running on that port? i.e. another process had grabbed the port making it unavailable to wgadmin? wgadmin is it's own web server. So it cannot share the port.

In terms of "wireguard not installed"... What OS and version are you using? I wonder if it's installed in a non-standard location when we check? We did not test every distribution of linux... 

Thanks for the typo.

Not sure why I only saw this notification now... sorry to keep you waiting.
